vendor.conf

```diff
@@ -13,6 +14,7 @@ github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76 # v1.1.0
 github.com/dgrijalva/jwt-go a2c85815a77d0f951e33ba4db5ae93629a1530af
 github.com/docker/distribution 83389a148052d74ac602f5f1d62f86ff2f3c4aa5
 github.com/docker/docker 6e3113f700dea1bf2785d94731b4b5a1e602d9ab
+github.com/docker/compose-on-kubernetes 939a5ba6c6dbfc78e6cc29b612716d5688b9a323
```

Comment:
Will we create a release for this? (I see there's a v0.4.16 tag https://github.com/docker/compose-on-kubernetes/releases but this PR probably needs changes that were merged after this https://github.com/docker/compose-on-kubernetes/compare/v0.4.16...939a5ba6c6dbfc78e6cc29b612716d5688b9a323)

Reply:
We will release a new version today yes, and then modify the sha to target it, thanks for catching that @thaJeztah 😄